cmd/starlark/starlark.go

```diff
@@ -120,9 +121,15 @@ func doMain() int {
 			return 1
 		}
 	case flag.NArg() == 0:
-		fmt.Println("Welcome to Starlark (go.starlark.net)")
+		fromStdin := !term.IsTerminal(int(os.Stdin.Fd()))
```

Comment:
The name and sense of the variable should be consistent. Let's use:
```
stdinIsTerminal := term.IsTerminal(int(os.Stdin.Fd()))
if stdinIsTerminal { ... }
```

Reply:
Sounds good, I'll amend that later on today.

Reply:
Sorry I ran out of time, I will try shortly.

Reply:
Done.